<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>624</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 54, 59);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>350</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>330</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(80, 138, 32);</string>
          </property>
          <property name="text">
           <string>TEMPERATURA ATUAL (°C)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_1">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>70</y>
            <width>180</width>
            <height>80</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>44</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(97, 99, 101);
background-color: rgb(35, 38, 41);
border-style: outset;
border-width: 1px;
border-radius: 7px;
border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>160</y>
            <width>350</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(80, 138, 32);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>HISTÓRICO DE MEDIÇÕES (°C)</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>220</y>
            <width>180</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>45</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(97, 99, 101);
background-color: rgb(35, 38, 41);
border-style: outset;
border-width: 1px;
border-radius: 7px;
border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>400</y>
            <width>180</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>45</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(97, 99, 101);
background-color: rgb(35, 38, 41);
border-style: outset;
border-width: 1px;
border-radius: 7px;
border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>85</x>
            <y>310</y>
            <width>180</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>45</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(97, 99, 101);
background-color: rgb(35, 38, 41);
border-style: outset;
border-width: 1px;
border-radius: 7px;
border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="bt_temp_at">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>31</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(69, 73, 78);
color: rgb(80, 138, 32);
border-style: solid;
border-width: 2px;
border-radius: 7px;
border-color: rgb(84, 87, 92);</string>
         </property>
         <property name="text">
          <string>TEMP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
